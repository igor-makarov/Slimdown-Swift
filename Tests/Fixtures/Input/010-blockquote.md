> A block quote
> across two lines.