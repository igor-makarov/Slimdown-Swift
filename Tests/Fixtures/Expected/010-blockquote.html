<blockquote>A block quote</blockquote>
<blockquote>across two lines.</blockquote>